tests/misc.js

```diff
@@ -159,6 +159,150 @@ describe('other toRDF tests', () => {
       done();
     });
   });
+
+  it.only('should handle list good @context @base', async () => {
+    const doc = {
+      "@context": {
+        "@base": "ex:",
+        "list": {
+          "@id": "foo:bar",
+          "@container": "@list",
+          "@type": "@id"
+        }
+      },
+      "list": [
+        "test"
+      ]
+    };
+    const rdf = await jsonld.toRDF(doc, {
+      format: 'application/n-quads'
+    });
+    assert.equal(
+      rdf,
+      '_:b0 <foo:bar> _:b1 .\n' +
+      '_:b1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <ex:test> .\n' +
+      '_:b1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> <http://www.w3.org/1999/02/22-rdf-syntax-ns#nil> .\n');
+  });
+
+  it.only('should handle list bad @context @base', async () => {
+    const doc = {
+      "@context": {
+        "@base": "::",
+        "list": {
```

Comment:
This isn't invalid, as ':' is a valid char, which is a component of segment-nz. When expanding relative to the document base, it becomes `https://w3c.github.io/json-ld-api/tests/toRdf/::`, which is valid.

I'll pick something else.